x-pack/test/detection_engine_api_integration/security_and_spaces/bundled_prebuilt_rules_package/prerelease_packages.ts

```diff
@@ -0,0 +1,66 @@
+/*
```

Comment:
I'm a little afraid of this test for the following reason: when installing rules, we check in the [Request Context what the Kibana Version and Kibana Branch are.](https://github.com/elastic/kibana/blob/main/x-pack/plugins/security_solution/server/lib/detection_engine/prebuilt_rules/api/install_prebuilt_rules_and_timelines/install_prebuilt_rules_package.ts). If they are different to a string containing `-SNAPSHOT` AND to `main`, respectively, then we assume we are not in a dev env and do not install the prerelease packages, only stable ones.

This has worked so far when running FTR, as those checks result in `isPrerelease` false.

Could this have any impact, though, once this PR has been merged to `main`? I mean, at that point the FTR tests will have already passed, and won't run again in an env in which the branch is `main`, but I'd really like a second opinion here.